src/components/packetdump.rs

```diff
@@ -516,7 +515,7 @@ impl PacketDump {
 
     pub fn get_arp_packages(&self) -> Vec<(DateTime<Local>, PacketsInfoTypesEnum)> {
         let a = &self.arp_packets.get_vec().to_vec();
-        a.clone()
+        a.to_vec()
```

Comment:
I would like to leave a clone here. This only double the memory on already created `vec`.